tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -29,6 +29,8 @@
 using Datadog.Trace.Sampling;
 using Datadog.Trace.Telemetry;
 using Datadog.Trace.Util.Http;
+using Datadog.Trace.VendoredMicrosoftCode.System.Reflection.Metadata;
+using Datadog.Trace.Vendors.dnlib.DotNet.Writer;
```

Comment:
Looks like these two lines were added by mistake. Mind reverting them?

Reply:
Thank you, removed them.